src/containers/Header/index.js

```diff
@@ -1 +1,14 @@
-export { default as Header } from './Header'
\ No newline at end of file
+import React from 'react'
+
+import { Logo } from '../../components'
```

Comment:
Future task:
Can you make it to be an absolute path like `from 'components'`
I think we have to do this in webpack config

Reply:
just make notes of these to do it later.